spec/sassdocs_helpers_spec.rb

```diff
@@ -13,10 +13,13 @@ def dothash(hash)
   before(:each) do
     allow(File).to receive(:read).and_return('{ "dependencies": { "govuk-frontend": { "version": "1.0.0" } } }')
     # Include mixin into a test class to allow us to mock File
+    # TODO Move constant definition
+    # rubocop:disable Lint/ConstantDefinitionInBlock
```

Comment:
Can this constant definition be edited?
https://msp-greg.github.io/rubocop/RuboCop/Cop/Lint/ConstantDefinitionInBlock.html

Reply:
Going to leave these as disabled for now.

Reply:
I was happy with these TODO comments by the way 👍